plumbing/protocol/packp/srvresp_test.go

```diff
@@ -0,0 +1,39 @@
+package packp
+
+import (
+	"bytes"
+
+	. "gopkg.in/check.v1"
+	"gopkg.in/src-d/go-git.v4/plumbing"
+)
+
+type ServerResponseSuite struct{}
```

Comment:
You could add at least 1 test for unexpected flush and 1 test for malformed input.